app/models/tta_activity_report.rb

```diff
@@ -57,6 +59,12 @@ def api
     ApiDelegator.use("tta", "activity_report", display_id: display_id, access_token: access_token)
   end
 
+  def check_display_id_format
+    unless display_id.match?(/^R\d{2}-AR-/)
```

Comment:
[comment, non-blocking] I can't find where I've seen the legacy display id format, but I think this regex will only work for the current format.  I think it's good enough for our current purposes, though.  Do you think it's worth adding a comment in case anybody is working from our code?

Reply:
sure